### PR TITLE
upgrading to ts

### PR DESCRIPTION
**Pull request by zikalino**

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+// your extension is activated the very first time the command is executed
+export function activate(context: vscode.ExtensionContext) {
+
+    // Use the console to output diagnostic information (console.log) and errors (console.error)
+    // This line of code will only be executed once when your extension is activated
+    console.log('Congratulations, your extension "vsc-extension-ansible" is now active!');
+
+    var outputChannel = vscode.window.createOutputChannel("VSCode extension for Ansible");
+    
+    // The command has been defined in the package.json file
```
Comment (yungez) [inline src/extension.ts]: remove those instruction comments from template?

File: src/test/extension.test.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+import * as vscode from 'vscode';
+import * as myExtension from '../extension';
+
+// Defines a Mocha test suite to group tests of similar kind together
+suite("Extension Tests", () => {
```
Comment (yungez) [inline src/test/extension.test.ts]: pls convert test cases.

**Comment by yungez** [comment]
@zikalino  HI Zim, pls fix travis.yml and appveyor.yml,  CI broken now.

File: src/test/extension.test.ts (hunk 2 above)
Comment (zikalino) [reply src/test/extension.test.ts]: ok

**Pull request merged**